app/src/main/java/org/jellyfin/androidtv/ui/shared/StrokeTextView.kt

```diff
@@ -0,0 +1,67 @@
+package org.jellyfin.androidtv.ui.shared
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.util.AttributeSet
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.core.content.ContextCompat
+import org.jellyfin.androidtv.R
+
+class StrokeTextView : AppCompatTextView {
+	private var strokeWidth = 0.0f
+	private var isDrawing: Boolean = false
+
+	constructor(
+		context: Context,
+		attrs: AttributeSet?,
+		defStyle: Int
+	) : super(context, attrs, defStyle) {
+		obtainStyledAttrs(context, attrs)
+	}
+
+	constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+		obtainStyledAttrs(context, attrs)
+	}
+
+	constructor(context: Context) : super(context) {
+		obtainStyledAttrs(context, attrs = null)
+	}
+
+	private fun obtainStyledAttrs(context: Context, attrs: AttributeSet?) {
```

Comment:
Obtaining the attributes should be similar to "ToolbarView". Get them in the `init{}` block.